links/templates/links/collection_detail.html

```diff
@@ -2,38 +2,71 @@
-                {% csrf_token %}
-                <button type="submit", name="link_id", value="{{ link.id }}">Remove Link</button>
-            </form>
+            <div class="links-item" title="{{link.website_name}}">
+                <img src="{{ link.image.url }}" alt="{{link.website_name}}">
+                <input class="toggle" type="checkbox" id="toggle_options{{forloop.counter}}"/>
+                <label class="option" for="toggle_options{{forloop.counter}}">{% trans "Options" %}</label>
+                <div class="option-grid">
+                    <a class="option" href="{% url 'deleteLink' link.id %}" >{% trans "Remove Link" %}</a>
+                    <a class="option" href="{% url 'uploadLink' link.id %}" >{% trans "Change Image" %}</a>
+                    {% if link.default %}
+                    <form action="{% url 'updateLink' %}" method="post">
+                        {% csrf_token %}
+                        <input class="option" type="submit" name="button" value="Remove Default">
+                        <input type="hidden" name="d_id" value={{link.id}}>
+                        <input type="hidden" name="set" value="Remove">
+                        <input type="hidden" name="type" value="Default">
+                    </form>
+                    {% else %}
+                    <form action="{% url 'updateLink' %}" method="post">
+                        {% csrf_token %}
+                        <input class="option" type="submit" name="button" value="Set Default">
+                        <input type="hidden" name="d_id" value={{link.id}}>
+                        <input type="hidden" name="set" value="Set">
+                        <input type="hidden" name="type" value="Default">
+                    </form>
+                    {% endif %}
+                    {% if link.enabled %}
+                    <form action="{% url 'updateLink' %}" method="post">
+                        {% csrf_token %}
+                        <input class="option" type="submit" name="button" value="Disable">
+                        <input type="hidden" name="d_id" value={{link.id}}>
+                        <input type="hidden" name="set" value="Disable">
+                        <input type="hidden" name="type" value="Enable">
+                    </form>
+                    {% else %}
+                    <form action="{% url 'updateLink' %}" method="post">
+                        {% csrf_token %}
+                        <input class="option" type="submit" name="button" value="Enable">
+                        <input type="hidden" name="d_id" value={{link.id}}>
+                        <input type="hidden" name="set" value="Set">
+                        <input type="hidden" name="type" value="Enable">
+                    </form>
+                    {% endif %}
+                    <label class="option" for="toggle_options{{forloop.counter}}">{% trans "Close Window" %}</label>
+                </div>
+            </div>
```

Comment:
On the collection detail page, users should not be able to change links to default or enable/disable them.

Reply:
Oh, my bad, I guess I copy/pasted and forgot to remove that. I'll take it out.

Reply:
Wait, I figured it out. That stuff was removed on the main branch, and this file doesn't have the updated changes. I don't why it didn't update when I merged the branches earlier. I'll still go ahead and fix it.